edgexfoundry/edgex-go
# feat: Secure MQTT Bus Credentials

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Use compose-builder on  https://github.com/joshua-silverio/edgex-compose/tree/secure-mqtt-bus to
 1. "make gen dev mqtt-bus <device or app service>
 3. docker-compose up to view logs of mqtt broker or device/app service 
 (should see mqtt broker connection w/ authmode:usernamepassword secretname: mqtt-bus"
## New Dependency Instructions (If applicable)
